libff/algebra/curves/mnt/mnt4/mnt4_init.cpp

```diff
@@ -17,9 +17,6 @@
 
 namespace libff {
 
-// bigint<mnt4_r_limbs> mnt4_modulus_r = mnt46_modulus_A;
```

Comment:
I find this comment helpful, just because keep track of which cycle is which is useful. 

I'll add an issue to make a better version of this comment than commented out code. (lets leave it deleted for this PR, thanks for flagging it!)